galasa-parent/dev.galasa.framework.api.cps/src/test/java/dev/galasa/framework/api/cps/internal/routes/TestAddPropertyInNamespaceRoute.java

```diff
@@ -0,0 +1,361 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.framework.api.cps.internal.routes;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.regex.Pattern;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.servlet.ServletOutputStream;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Null;
+
+import org.junit.Test;
+
+import dev.galasa.framework.api.common.mocks.MockIConfigurationPropertyStoreService;
+import dev.galasa.framework.api.cps.internal.CpsServletTest;
+import dev.galasa.framework.api.cps.internal.mocks.MockCpsServlet;
+import dev.galasa.framework.spi.ConfigurationPropertyStoreException;
+
+public class TestAddPropertyInNamespaceRoute extends CpsServletTest {
+
+    /*
+     * Regex Path
+     */
+
+    @Test
+    public void TestPathRegexExpectedPathReturnsTrue(){
```

Comment:
Missing `throws Exception` in these tests (otherwise we won't know if they fail)